lib/config/index.js

```diff
@@ -12,4 +30,7 @@ nconf.argv()
     });
 
 nconf.defaults(defaults);
-module.exports = nconf;
\ No newline at end of file
+
+// @TODO: this file gets cached after the first require, we need to redesign how we load the config
+// @TODO: for example: require('ghost-ignition').config()
```

Comment:
@ErisDS with this PR we can fix the issue which was raised, but we need to add a design change (which needs to be adapted everywhere `ghost-ignition` is used). if module A uses `ghost-ignition` it requires the module and afterwards its cached. so if module B is then requiring the module, the config is already cached.
but we can add after release 0.10.1, but tell me what do you think!